docker/openproblems-r-extras/README.md

```diff
@@ -11,7 +11,9 @@ R: 3.6
 R packages:
 
 * batchelor
+* rsvd
 
 Python packages:
 
+* scprep
```

Comment:
Already included in `openproblems`, not necessary here.

Reply:
Correct! I forgot scprep was already sourced in openproblems